qiskit/assembler/assemble_schedules.py

```diff
@@ -13,33 +13,21 @@
 # that they have been altered from the originals.
 
 """Assemble function for converting a list of circuits into a qobj."""
-from collections import defaultdict
-from typing import Any, Dict, List, Tuple
-import hashlib
 
-from qiskit.exceptions import QiskitError
-from qiskit.pulse import Schedule, Acquire, Delay, Play, transforms
-from qiskit.pulse.pulse_lib import ParametricPulse, SamplePulse
-from qiskit.pulse.commands import (Command, PulseInstruction, AcquireInstruction,
-                                   DelayInstruction, ParametricInstruction)
-from qiskit.qobj import (PulseQobj, QobjHeader, QobjExperimentHeader,
-                         PulseQobjInstruction, PulseQobjExperimentConfig,
-                         PulseQobjExperiment, PulseQobjConfig, PulseLibraryItem)
-from qiskit.qobj.converters import InstructionToQobjConverter, LoConfigConverter
-from qiskit.qobj.converters.pulse_instruction import ParametricPulseShapes
-from qiskit.qobj.utils import MeasLevel, MeasReturnType
+from qiskit import qobj, pulse
+from qiskit.pulse import lowering
 
 from .run_config import RunConfig
 
 
-def assemble_schedules(schedules: List[Schedule],
+def assemble_schedules(program: pulse.Program,
```

Comment:
If this is user facing, `schedules` needs to be deprecated, if it's not, this file can be removed.

Reply:
`assemble_schedules` is not userfacing. The documented entry point is `assemble` as this is what constructs the `run_config`.